migrations/2018-05-14-200933_setup/up.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE album (
+  id VARCHAR(36) PRIMARY KEY NOT NULL,
+  artwork_url TEXT,
+  name TEXT NOT NULL,
+  artist_id VARCHAR(36) NOT NULL REFERENCES artist(id),
+  release_year INTEGER NOT NULL,
+  time_added INTEGER NOT NULL,
+
```

Comment:
Extra line

Reply:
The stuff below this line is user stats. It will be moved eventually. That's why there's a seperation.